apparmor.d/profiles-m-r/qbittorrent

```diff
@@ -31,11 +31,13 @@ profile qbittorrent @{exec_path} {
   include <abstractions/dbus-accessibility-strict>
   include <abstractions/dbus-network-manager-strict>
   include <abstractions/dbus-gtk>
+  include <abstractions/dri-common>
   include <abstractions/dri-enumerate>
   include <abstractions/wayland>
   include <abstractions/mesa>
   include <abstractions/openssl>
   include <abstractions/ssl_certs>
+  include <abstractions/nvidia>
```

Comment:
We should start to think about something like `gpu` abstraction.

Reply:
`abstractions/opencl` is pretty mucj a gpu abstraction already.

Reply:
Close, but to broad for my liking.